client/checkout/woopay/express-button/woopay-express-checkout-payment-method.js

```diff
@@ -18,15 +24,27 @@ const api = new WCPayAPI(
 	request
 );
 
+const WooPayExpressCheckoutButtonContainer = () => {
+	const onRefChange = useCallback( ( node ) => {
+		if ( node ) {
```

Comment:
I wasn't able to get into a scenario where node was empty, I was wondering if there is a process that could trigger that just to confirm what happens when this condition is false?

Reply:
When that condition is `false`, the WooPay button isn't rendered. For this scenario to happen there must be something really wrong happening with React, so I'd say it's nearly impossible. We could totally remove the `if`, however, there's no performance implication and I think we can afford having this check to walk on the safe side.

Reply:
> For this scenario to happen there must be something really wrong happening with React

Awesome, in that case no concern from my side and having the check is definitely worth it, just wanted to make sure I wasn't missing an obvious edge case to test while reviewing thanks!